apps/templates/resource_download.html

```diff
@@ -50,10 +50,10 @@ <h2>{% trans 'Windows Remote application publisher tools' %}</h2>
     </div>
 
     <div class="group">
-        <h2>JumpServer {% trans 'Offline video player' %} v0.1.9</h2>
+        <h2>JumpServer {% trans 'Offline video player' %} v{{ VIDEO_PLAYER_VERSION }}</h2>
         <ul>
-            <li><a href="/download/public/JumpServer.Video.Player-{{ VIDEO_PLAYER_VERSION }}.dmg">jumpserver-video-player.dmg</a></li>
-            <li><a href="/download/public/JumpServer.Video.Player.Setup.{{ VIDEO_PLAYER_VERSION }}.exe">jumpserver-video-player.exe</a></li>
+            <li><a href="/download/public/JumpServerVideoPlayer-{{ VIDEO_PLAYER_VERSION }}.dmg">jumpserver-video-player.dmg</a></li>
+            <li><a href="/download/public/JumpServerVideoPlayer-{{ VIDEO_PLAYER_VERSION }}.exe">jumpserver-video-player.exe</a></li>
         </ul>
     </div>
 </div>
```

Comment:
There are no known issues with this code snippet regarding functionality or efficiency. However, I suggest making sure that you always refer to your API documentation while using resources such as official downloads and releases. It's also recommended to run checks on file paths (e.g., double-checking the URLs) and ensure they meet your system's access policies before downloading anything from them.